subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -1291,6 +1292,14 @@ static void ticker_cb(uint32_t ticks_at_expire, uint32_t ticks_drift,
 			     &mfy_lll_prepare);
 	LL_ASSERT(!ret);
 
+	/* Calculate the BIG reference point of current BIG event */
+	remainder_us = remainder;
```

Comment:
Is remainder in microseconds?

Reply:
Remainder is in HAL specific unit (+/-30.517 us stored in picoseconds to fit 32-bits). `hal_ticker_remove_jitter()`  is used to get a positive remainder in microseconds by adjusting the `ticks_to_expire` accordingly.